js/widgets/forms/textinput.js

```diff
@@ -44,21 +47,16 @@ $.widget( "mobile.textinput", {
 
 		var options = this.options,
 			isSearch = this.element.is( "[type='search'], :jqmData(type='search')" ),
-			isTextarea = this.element[ 0 ].tagName === "TEXTAREA",
-			isRange = this.element.is( "[data-" + ( $.mobile.ns || "" ) + "type='range']" ),
 			inputNeedsWrap = ( (this.element.is( "input" ) ||
 				this.element.is( "[data-" + ( $.mobile.ns || "" ) + "type='search']" ) ) &&
-					!isRange );
+					!this.element.is( "[data-" + ( $.mobile.ns || "" ) + "type='range']" ) );
```

Comment:
I'm thinking we should handle this in range slider not in textinput. like not use the textinput widget at all and just use the classes.